### PR TITLE
Odroid_C2/u-boot: move to project specific package

### PR DESCRIPTION
**Pull request by Raybuntu**
As requested. Splitting out C2 u-boot fork out to be a project specific package. I've tested a C2 build and it seems to work.
I've not tested imx so I hope I did not break it.

Ping @lrusak

### CODE REVIEW

File: packages/tools/u-boot/package.mk [hunk 1]
```diff
@@ -1,19 +1,20 @@
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
+#      This file is part of LibreELEC - http://www.libreelec.tv
```
Comment (CGarces) [inline packages/tools/u-boot/package.mk]: This file is part of LibreELEC - https://libreelec.tv

Comment (Raybuntu) [reply packages/tools/u-boot/package.mk]: Thanks. Fixed.

File: packages/tools/u-boot/package.mk [hunk 2]
```diff
@@ -1,19 +1,20 @@
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
+#      This file is part of LibreELEC - https://www.libreelec.tv
```
Comment (CvH) [inline packages/tools/u-boot/package.mk]: `https://www.libreelec.tv` -> `https://libreelec.tv`

Comment (Raybuntu) [reply packages/tools/u-boot/package.mk]: And Thanks again.

**Comment by vpeter4** [comment]
Interesting how things gets around: I proposed project u-boot package long time back for imx6 not to mess with "system" package and make it more complex. But no, this was not allowed. And now we see exactly this. Funny.

Anyway, you can remove loop around UBOOT_CONFIG because it is used when building same u-boot package with different configs. But you are building only one.

**Comment by Raybuntu** [comment]
Thank you @vpeter4. Totally missed that. I fixed it.
I'm not sure this PR will be allowed either but I'm giving it a shot :smile:

**Comment by lrusak** [comment]
I would rather have it exist next to u-boot. Project specific package directories are meh.

I'd rather have it be `packages/tools/u-boot-hardkernel`

**Comment by Raybuntu** [comment]
Why so? I don't think that anything else will be using u-boot-hardkernel

**Comment by lrusak** [comment]
I never liked having packages exist outside of the packages/ directory.

Anyways. I'm fine with merging it as it's better than before

**Pull request merged**